src/amo/components/Header/index.js

```diff
@@ -0,0 +1,93 @@
+import React, { PropTypes } from 'react';
+import { compose } from 'redux';
+import { connect } from 'react-redux';
+
+import Link from 'amo/components/Link';
+import SearchForm from 'amo/components/SearchForm';
+import AuthenticateButton from 'core/components/AuthenticateButton';
+import translate from 'core/i18n/translate';
+import Button from 'ui/components/Button';
+import Icon from 'ui/components/Icon';
+
+import 'mozilla-tabzilla/css/_tabzilla.scss';
+import './styles.scss';
+
+
+export class HeaderBase extends React.Component {
+  static propTypes = {
+    i18n: PropTypes.object.isRequired,
+    isHomePage: PropTypes.bool.isRequired,
+    lang: PropTypes.string.isRequired,
+    location: PropTypes.object.isRequired,
+    SearchFormComponent: PropTypes.node.isRequired,
+    query: PropTypes.string,
+  }
+
+  static defaultPropTypes = {
+    SearchFormComponent: SearchForm,
+    isHomePage: false,
+  }
+
+  render() {
+    const {
+      SearchFormComponent,
+      i18n,
+      isHomePage,
+      lang,
+      location,
+      query,
+    } = this.props;
+    const headerLink = (
+      <Link className="Header-title" to="/">
+        <Icon className="Header-addons-icon" name="extensions-complex" />
+        {i18n.gettext('Firefox Add-ons')}
```

Comment:
Would be good to add a comment for l10ners about not translating "Firefox". This test in the extaction shows what is looked for: https://github.com/mozilla/babel-gettext-extractor/blob/master/test/test.js#L55